spec/factories/like_factory.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+
+  factory :like do
+
```

Comment:
A user shouldn't have a like without liking something. Add presence validation to like model for liking_type and liking_id